assignments1/calculateFrequency.js

```diff
@@ -0,0 +1,16 @@
+function calculateFrequency(string) {
+  
+  var freq={};
+  for(var iter=0;iter<string.length;iter++)
+    {
+         var ch=string.charAt(iter);
+      if(ch>='a'&&ch<='z')
```

Comment:
spaces before and after operator is required always
eg. instead of `a>b` , write `a > b`